Makefile

```diff
@@ -48,7 +44,6 @@ smoke-test:
 	BCDA_SSAS_CLIENT_ID=$(SSAS_ADMIN_CLIENT_ID) BCDA_SSAS_SECRET=$(SSAS_ADMIN_CLIENT_SECRET) CLIENT_ID=$(CLIENT_ID) CLIENT_SECRET=$(CLIENT_SECRET) docker-compose -f docker-compose.test.yml run --rm -w /go/src/github.com/CMSgov/bcda-app/test/smoke_test tests sh smoke_test.sh
```

Comment:
Is `BCDA_SSAS_CLIENT_ID` and `BCDA_SSAS_SECRET` used in the standard smoke test?  I didn't think it was needed.

Reply:
Good catch.  My initial tests seemed to show I should leave this line alone, but everything is fine now.  😄 Removed in latest commit.